src/app/component/quiz/quiz.component.html

```diff
@@ -1,108 +1,146 @@
-        [showNavigationIndicators]="false"
-        [interval]="0"
-        [keyboard]="false"
+<form [formGroup]="quizForm">
+  <div formArrayName="form">
+    <ngb-carousel
+      #carousel
+      *ngIf="question?.length"
+      [showNavigationArrows]="false"
+      [showNavigationIndicators]="false"
+      [interval]="0"
+      [keyboard]="false"
+    >
+      <ng-template
+        ngbSlide
+        *ngFor="let arrayloop of FormArray.controls; let i = index"
       >
-        <ng-template
-          ngbSlide
-          *ngFor="let arrayloop of FormArray.controls; let i = index"
-        >
-          <ng-container [formGroupName]="i">
-            <div class="p-5">
-              <div class="card card-layout">
-                <div class="card-body card-body-layout">
-                  <div class="quiz">
-                    <div class="data">
+        <ng-container [formGroupName]="i">
+          <div class="container-fluid">
+            <div class="d-flex align-items-center min-vh-100">
+              <div class="container mx-auto">
+                <div class="card quiz_card h-auto">
+                  <div
+                    class="card-header header_txt d-flex align-items-center justify-content-between position-relative"
+                  >
+                    <div class="header_title user-select-none">
+                      {{ [this.selectedQuizType] }}
+                    </div>
+                    <div
+                      class="timer d-flex align-items-center justify-content-between rounded-2"
+                    >
+                      <div class="timer_left_txt user-select-none">
+                        Time Left
+                      </div>
+                      <div
+                        class="timer_sec user-select-none text-center rounded-2"
+                      >
+                        {{ FormArray.controls.at(i)?.value?.timer }}
+                      </div>
+                    </div>
+                  </div>
+
+                  <div class="card-body que_txt d-inline-block rounded-2">
+                    <div class="que_data user-select-none">
                       {{ "Q-" + (i + 1) }} {{ question[i]["question"] }}
                     </div>
-                    <div class="timer">
-                      <i class="fa fa-clock timer"
-                        >&#xf017;
-                        {{ FormArray.controls.at(i)?.value?.timer }} sec</i
+
+                    <div class="option_list">
+                      <ng-container
+                        *ngFor="
+                          let card of question[this.questionIndex].options
+                        "
                       >
+                        <div class="form_check">
+                          <label
+                            class="form-check-label option w-100 rounded-2 align-items-center user-select-none"
+                            *ngIf="card.type == 'checkbox'"
+                            [ngClass]="
```

Comment:
can we check this condition like 
 [class.hightlight-selected]="FormArray.controls.at(i)?.value.radioValue ===card.value"